giskard/scanner/issues.py

```diff
@@ -25,6 +26,42 @@ class IssueGroup:
     description: str
 
 
+class ScanExamples(ABC):
+    """
+    Abstract class to manage examples from different data types.
+    """
+
+    @abstractmethod
+    def add_examples(self, example: Any):
```

Comment:
naming must be consistent between function & argument (is it one or several arguments we are adding ?)

if it's several, i would go with `extend`to copy existing naming in python

Reply:
Also, missing doc string